Makefile

```diff
@@ -653,7 +653,7 @@ test-cli-local: $(CLI_BIN)
 
 .PHONY: test-cli-local-mutate
 test-cli-local-mutate: $(CLI_BIN)
-	@$(CLI_BIN) test ./test/cli/test-mutate
+	# @$(CLI_BIN) test ./test/cli/test-mutate
```

Comment:
This is not disabled in main🤔

Reply:
Yep, will be enabled again in another cherry pick.

Reply:
We had to do that in this PR because the lazy loading implementation is not correct yet (until jim's pr fixes it, last one that will be cherry picked).

Reply:
Okay.